tests/phpunit/test-wc-stripe-helper.php

```diff
@@ -394,9 +394,9 @@ public function provide_is_wallet_payment_method(): array {
 	 * @return void
 	 */
 	public function test_handle_main_stripe_settings() {
-		WC_Stripe_Helper::update_main_stripe_settings( [ 'test' => 'test' ] );
+		WC_Stripe_Helper::update_main_stripe_settings( [ 'test' => 'abc' ] );
 		$current_settings = WC_Stripe_Helper::get_stripe_settings();
-		$this->assertSame( [ 'test' => 'test' ], $current_settings );
+		$this->assertSame( $current_settings['test'], 'abc' );
```

Comment:
I updated this to take into account hooks on `pre_update_option_woocommerce_stripe_settings` that may inject extra settings.